docs/usage-telemetry.md

```diff
@@ -0,0 +1,105 @@
+# Usage telemetry
+
+Starting with version 23.10 LTS, EventStoreDB has introduced a telemetry feature. This collects anonymous, aggregated usage statistics and sends them periodically to Event Store. Telemetry data helps us refine and improve our product based on real usage patterns.
+
+## What is usage telemetry
+
+EventStoreDB telemetry only tracks non-Personally-Identifiable Information. Collected data does not allow Event Store to fingerprint the users by any of the collected data points.
```

Comment:
Fingeprint users _and_ company ?
we have corporate customers

Reply:
I'd also add that in _no_ way we track any _data_ like stream name or events paylod & metadata

Reply:
It's mentioned down below.